asv_bench/benchmarks/benchmarks_hindcast.py

```diff
@@ -0,0 +1,166 @@
+# Write the benchmarking functions here.
+# See "Writing benchmarks" in the asv docs for more information.
+
+
+import dask
+import numpy as np
+import xarray as xr
+
+from climpred.bootstrap import bootstrap_hindcast
+# from climpred.constants import HINDCAST_COMPARISONS
+from climpred.prediction import compute_hindcast
+
+from . import parameterized, randn, requires_dask
+
+HINDCAST_COMPARISONS = ['m2r']  # e2r and probabilistic dont match
+METRICS = ['rmse', 'pearson_r', 'crpss']
+
+bootstrap = 8
+
+
+def _ensure_loaded(res):
+    """Compute no lazy results."""
+    if dask.is_dask_collection(res):
+        res = res.compute()
+    return res
+
+
+class Generate:
+    """
+    Generate random hind, control to be benckmarked.
+    """
+
+    timeout = 600
+    repeat = (2, 5, 20)
+
+    def make_hind_ref(self):
+        """hind and ref mimik hindcast experiment"""
```

Comment:
Add proper docstring here (don't need to document `self`) but can have

```python
"""Generates initialized hindcast and observational data, mimicking a hindcast experiment.

Attributes:
    hind (xarray Dataset): ...
    observations (xarray Dataset): ...
"""
```

Reply:
Also add `hist` to this since it's added here. I also prefer `uninit` or `forced` or something. I don't think `hist` is very descriptive.

Reply:
I wont document the attributes. this is not needed anywhere else